internal/metrics/metrics.go

```diff
@@ -0,0 +1,38 @@
+package metrics
+
+import (
+	"sync"
+
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+// https://prometheus.io/docs/practices/naming/
+var (
+	ClickHouseBatchSize = prometheus.NewGauge(prometheus.GaugeOpts{
```

Comment:
I am not sure if this is a useful metric. Who runs Jaeger ultimately knows what is the setting. 

The same applies to ClickHouseFlushInterval.